src/stcal/ramp_fitting/ols_fit.py

```diff
@@ -645,6 +655,9 @@ def ols_ramp_fit_single(ramp_data, buffsize, save_opt, readnoise_2d, gain_2d, we
     weighting : str
         'optimal' is the only valid value
 
+    avg_dark_current : float (electrons)
```

Comment:
Remove these lines, because it's no longer passed as a parameter.

Reply:
You're right. I removed it.